src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -101,6 +104,23 @@ const OrganizationStream = createReactClass({
     if (!this.state.loading) {
       this.fetchData();
       fetchTags(this.props.organization.slug);
+
+      fetchOrgMembers(this.api, this.props.organization.slug).then(members => {
+        members = members.filter(m => m.user);
+        // Update the store, as avatars rely on them.
+        MemberListStore.loadInitialData(members.map(m => m.user));
```

Comment:
I think updating stores is generally done in the action creator, the grouping code below as well if you'd like to extract it from this component